core/src/main/java/org/elasticsearch/action/index/IndexResponse.java

```diff
@@ -110,12 +70,23 @@ public void writeTo(StreamOutput out) throws IOException {
     public String toString() {
```

Comment:
why is there only a toString() here, but not in DeleteResponse and UpdateResponse?